{"timestamp":"2025-07-13T14:15:07.796620","level":"info","event":"DAG bundles loaded: dags-folder","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-07-13T14:15:07.797630","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/activefence.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-07-13T14:15:11.467351","level":"info","event":"Columns :      article_id  ...              load_timestamp\n0  cql6vgwyv4xo  ...  2025-07-13T14:15:09.417632\n1  c79ewz7qej4o  ...  2025-07-13T14:15:09.346872\n2  c5ylkyrkjnzo  ...  2025-07-13T14:15:09.484704\n3      w3ct6zns  ...  2025-07-13T14:15:09.478878\n4  cz9y1l397vno  ...  2025-07-13T14:15:09.423363\n\n[5 rows x 5 columns]","logger":"unusual_prefix_0aab8e1abe4a8f324d874f9a30de87ad63865522_activefence"}
{"timestamp":"2025-07-13T14:15:11.491665Z","level":"error","event":"[2025-07-13T14:15:11Z WARN  aws_config::imds::region] failed to load region from IMDS err=failed to load IMDS session token: dispatch failure: other: client error (Connect): tcp connect error: Connection refused (os error 111) (FailedToLoadToken(FailedToLoadToken { source: DispatchFailure(DispatchFailure { source: ConnectorError { kind: Other(None), source: hyper_util::client::legacy::Error(Connect, ConnectError(\"tcp connect error\", 169.254.169.254:80, Os { code: 111, kind: ConnectionRefused, message: \"Connection refused\" })), connection: Unknown } }) }))","chan":"stderr","logger":"task"}
{"timestamp":"2025-07-13T14:15:11.494802Z","level":"error","event":"[2025-07-13T14:15:11Z WARN  aws_config::imds::region] failed to load region from IMDS err=failed to load IMDS session token: dispatch failure: other: client error (Connect): tcp connect error: Connection refused (os error 111) (FailedToLoadToken(FailedToLoadToken { source: DispatchFailure(DispatchFailure { source: ConnectorError { kind: Other(None), source: hyper_util::client::legacy::Error(Connect, ConnectError(\"tcp connect error\", 169.254.169.254:80, Os { code: 111, kind: ConnectionRefused, message: \"Connection refused\" })), connection: Unknown } }) }))","chan":"stderr","logger":"task"}
{"timestamp":"2025-07-13T14:15:11.498045Z","level":"error","event":"[2025-07-13T14:15:11Z WARN  aws_config::imds::region] failed to load region from IMDS err=failed to load IMDS session token: dispatch failure: other: client error (Connect): tcp connect error: Connection refused (os error 111) (FailedToLoadToken(FailedToLoadToken { source: DispatchFailure(DispatchFailure { source: ConnectorError { kind: Other(None), source: hyper_util::client::legacy::Error(Connect, ConnectError(\"tcp connect error\", 169.254.169.254:80, Os { code: 111, kind: ConnectionRefused, message: \"Connection refused\" })), connection: Unknown } }) }))","chan":"stderr","logger":"task"}
{"timestamp":"2025-07-13T14:15:11.505319Z","level":"error","event":"[2025-07-13T14:15:11Z WARN  aws_config::imds::region] failed to load region from IMDS err=failed to load IMDS session token: dispatch failure: other: client error (Connect): tcp connect error: Connection refused (os error 111) (FailedToLoadToken(FailedToLoadToken { source: DispatchFailure(DispatchFailure { source: ConnectorError { kind: Other(None), source: hyper_util::client::legacy::Error(Connect, ConnectError(\"tcp connect error\", 169.254.169.254:80, Os { code: 111, kind: ConnectionRefused, message: \"Connection refused\" })), connection: Unknown } }) }))","chan":"stderr","logger":"task"}
{"timestamp":"2025-07-13T14:15:11.508565Z","level":"error","event":"[2025-07-13T14:15:11Z WARN  aws_config::imds::region] failed to load region from IMDS err=failed to load IMDS session token: dispatch failure: other: client error (Connect): tcp connect error: Connection refused (os error 111) (FailedToLoadToken(FailedToLoadToken { source: DispatchFailure(DispatchFailure { source: ConnectorError { kind: Other(None), source: hyper_util::client::legacy::Error(Connect, ConnectError(\"tcp connect error\", 169.254.169.254:80, Os { code: 111, kind: ConnectionRefused, message: \"Connection refused\" })), connection: Unknown } }) }))","chan":"stderr","logger":"task"}
{"timestamp":"2025-07-13T14:15:11.512203Z","level":"error","event":"[2025-07-13T14:15:11Z WARN  aws_config::imds::region] failed to load region from IMDS err=failed to load IMDS session token: dispatch failure: other: client error (Connect): tcp connect error: Connection refused (os error 111) (FailedToLoadToken(FailedToLoadToken { source: DispatchFailure(DispatchFailure { source: ConnectorError { kind: Other(None), source: hyper_util::client::legacy::Error(Connect, ConnectError(\"tcp connect error\", 169.254.169.254:80, Os { code: 111, kind: ConnectionRefused, message: \"Connection refused\" })), connection: Unknown } }) }))","chan":"stderr","logger":"task"}
{"timestamp":"2025-07-13T14:15:11.516460","level":"error","event":"Bronze processing failed: Operation not supported\n          \u001b[31m↳\u001b[0m the credential provider was not enabled\n           \u001b[31m↳\u001b[0m no providers in chain provided credentials\n","logger":"unusual_prefix_0aab8e1abe4a8f324d874f9a30de87ad63865522_activefence"}
{"timestamp":"2025-07-13T14:15:11.516882","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"OSError","exc_value":"Operation not supported\n          \u001b[31m↳\u001b[0m the credential provider was not enabled\n           \u001b[31m↳\u001b[0m no providers in chain provided credentials\n","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":867,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1159,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":216,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":239,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/dags/activefence.py","lineno":143,"name":"process_bronze"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/deltalake/writer/writer.py","lineno":106,"name":"write_deltalake"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/deltalake/writer/_utils.py","lineno":32,"name":"try_get_table_and_table_uri"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/deltalake/writer/_utils.py","lineno":47,"name":"try_get_deltatable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/deltalake/table.py","lineno":161,"name":"__init__"}],"is_group":false,"exceptions":[]}]}
